### PR TITLE
Add Windows support to openmp_examples tests

### PR DESCRIPTION
**Pull request by tdusnoki**

### CODE REVIEW

**Review by bryanpkc — APPROVED** [review]

File: test/openmp_examples/lit/Example_SIMD.1f.sh [hunk 1]
```diff
@@ -5,5 +5,5 @@
 
 # Shared lit script for each tests. Run bash commands that run tests with make.
 
-# RUN: KEEP_FILES=%keep FLAGS=%flags TEST_SRC=%s MAKE_FILE_DIR=%S/.. bash %S/runmake | tee %t 
+# RUN: env KEEP_FILES=%keep FLAGS=%flags TEST_SRC=%/s MAKE_FILE_DIR=%/S/.. bash %/S/runmake | tee %/t
```
Comment (pawosm-arm) [inline test/openmp_examples/lit/Example_SIMD.1f.sh:8]: what are those `%/`?

Comment (bryanpkc) [reply test/openmp_examples/lit/Example_SIMD.1f.sh:8]: From https://llvm.org/docs/CommandGuide/lit.html#substitutions :

Comment (pawosm-arm) [reply test/openmp_examples/lit/Example_SIMD.1f.sh:8]: I presume this couldn't be done any nicer way...

**Review by pawosm-arm — APPROVED** [review]

File: test/openmp_examples/lit/Example_SIMD.1f.sh (hunk 1 above)
Comment (kaadam) [reply test/openmp_examples/lit/Example_SIMD.1f.sh:8]: Yeah,  mingw64 bash is a bit picky about path. Unfortunately, We haven't found other way how we can force Posix slashes on Windows. that's why we decided to use predefined lit substitution. Despite it results really huge changes in tests.

**Pull request merged**